### PR TITLE
Chapter 6: results

### PR DESCRIPTION
**Pull request by LeidongX**

### CODE REVIEW

File: tex/chapters/chapter6.tex [hunk 1]
```diff
@@ -0,0 +1,399 @@
...
+ \begin{split}
+  -\nabla D(\mathbf{r})_1 \nabla \phi_1 (\mathbf{r}) + \Sigma_{r1}(\mathbf{r}) \phi_1 (\mathbf{r}) 
+    &= \frac{1}{k} \left (\nu\Sigma_{f1}\phi_1(\mathbf{r}) + \nu\Sigma_{f2}\phi_2(\mathbf{r}) \right ) \\
+ -\nabla D(\mathbf{r})_2 \nabla \phi_2 (\mathbf{r}) + \Sigma_{a2}(\mathbf{r}) \phi_2 (\mathbf{r}) 
+    &= \Sigma_{s1\to 2}\phi_1(\mathbf{r}) \, ,
+ \end{split}
+ \label{eq:twogroupdiff}
+\end{equation}
+where the notations are consistent as \CHAPTER{chapter:multigroup}.
+All parameters including the cross sections are defined in the technical report published from Argonne national laboratory\cite{center1977benchmark}.
```
Comment (RLReed) [inline tex/chapters/chapter6.tex]:
```suggestion
All parameters including the cross sections are defined in the technical report published from Argonne National Laboratory\cite{center1977benchmark}.
```

File: tex/chapters/chapter6.tex [hunk 2]
```diff
@@ -0,0 +1,399 @@
...
+The basic core layout is shown in \FIGURE{fig:IAEA},
```
Comment (RLReed) [inline tex/chapters/chapter6.tex]:
```suggestion
The basic core layout is shown in \FIGURE{fig:IAEA}, where the west and south boundary are reflective, and the north and east boundary are vacuum.
```

File: tex/chapters/chapter6.tex [hunk 3]
```diff
@@ -0,0 +1,399 @@
...
+
+\begin{figure*}[htb!]
```
Comment (RLReed) [inline tex/chapters/chapter6.tex]: You may consider using the standalone package that will separate out the tex pictures into a separate document.  You can see my dissertation for the right code.

File: tex/chapters/chapter6.tex [hunk 4]
```diff
@@ -0,0 +1,399 @@
...
+  \foreach \y/\ytext in {0, 10, 70, 90, 130, 150, 170}
+      \draw[black,yshift=0.1*\y cm] (.3,0) -- (0,0) node[left] {$\ytext$};
+
+\end{scope}
+\end{tikzpicture}
+\caption{Geometry as modeled for the IAEA 2-D diffusion benchmark.  Material properties can be found in the benchmark documentation.  Materials 0 and 1 are fuel, material 2 represents control, while material 3 represents the outer reflector.}
+\label{fig:iaea2d}
+\end{figure*}
+
+where the west and south boundary are reflect, and the north and east boundary are vaccum.
```
Comment (RLReed) [inline tex/chapters/chapter6.tex]:
```suggestion
```

File: tex/chapters/chapter6.tex [hunk 5]
```diff
@@ -0,0 +1,399 @@
...
+The mesh-center, finite-volume approximation was employed on a uniform, 45 $\times$ 45 spatial mesh.
+The discrete ordinates transport code DETRAN was used to generate the explicit system matrix $\mathbf{A}$ as a .out file.
```
Comment (RLReed) [inline tex/chapters/chapter6.tex]:
```suggestion
The discrete ordinates transport code DETRAN was used to generate the explicit system matrix $\mathbf{A}$.
```

File: tex/chapters/chapter6.tex [hunk 6]
```diff
@@ -0,0 +1,399 @@
...
+Upon discretization, the entire set of equations was cast in terms of the fission source density, i.e., $\mathbf{f} = \nu\Sigma_{f1}\phi_1(\mathbf{r}) + \nu\Sigma_{f2}\phi_2(\mathbf{r})$, which results in a $2025 \times 2025$ operator. 
+Hence, problem(1) is not a relatively large one, but it proved to be a valuable test case for the method, and, therefore, ensure a reasonable computing time while debugging the code.
```
Comment (RLReed) [inline tex/chapters/chapter6.tex]:
```suggestion
Hence, problem(1) is not large, but it proved to be a valuable test case for the method, and, therefore, ensured a reasonable computing time while debugging the code.
```

File: tex/chapters/chapter6.tex [hunk 7]
```diff
@@ -0,0 +1,399 @@
...
+
+Note that the eigenvalue is typically updated by fission rate.
+At the time of the study, the in-house transport solver used the ratios of fluxes for the eigenvalue update.
```
Comment (RLReed) [inline tex/chapters/chapter6.tex]:
```suggestion
At the time of the study, the in-house transport solver used the ratio of fluxes for the eigenvalue update.
```

File: tex/chapters/chapter6.tex [hunk 8]
```diff
@@ -0,0 +1,399 @@
...
+However, the code has since been updated to use the ratio of fission densities, and found a negligible difference in results.
+
+All calculations were starting with a vector of which each element was sampled from the uniform distribution $U[0, 1]$.
```
Comment (RLReed) [inline tex/chapters/chapter6.tex]:
```suggestion
All calculations were initialized with a vector in which each element was sampled from the uniform distribution $U[0, 1]$. 
```

File: tex/chapters/chapter6.tex [hunk 9]
```diff
@@ -0,0 +1,399 @@
...
+Upon discretization, the entire set of equations was cast in terms of the fission source density, i.e., $\mathbf{f} = \nu\Sigma_{f1}\phi_1(\mathbf{r}) + \nu\Sigma_{f2}\phi_2(\mathbf{r})$, which results in a $2025 \times 2025$ operator. 
+Hence, problem(1) is not a relatively large one, but it proved to be a valuable test case for the method, and, therefore, ensure a reasonable computing time while debugging the code.
+
+Note that the eigenvalue is typically updated by fission rate.
+At the time of the study, the in-house transport solver used the ratios of fluxes for the eigenvalue update.
+However, the code has since been updated to use the ratio of fission densities, and found a negligible difference in results.
+
+All calculations were starting with a vector of which each element was sampled from the uniform distribution $U[0, 1]$. 
+This randomized starting vector helps to ensure that all eigenmodes can be present.
+A formal sensitivity study was not performed to understand how this initial guess impacts the algorithm performance, but scoping studies suggest there is little impact on the numbers of iterations required for any particular algorithm.
```
Comment (RLReed) [inline tex/chapters/chapter6.tex]:
```suggestion
A formal sensitivity study was not performed to understand how this initial guess impacts the algorithm performance, but scoping studies suggest there is little impact on the number of iterations required for any particular algorithm.
```

File: tex/chapters/chapter6.tex [hunk 10]
```diff
@@ -0,0 +1,399 @@
...
+In this case, a reference solution was computed using the implicitly-restarted Arnoldi method as implemented in SciPy \cite{scipy}.
+All DMD calculations were performed using the Python package {\tt PyDMD} \cite{pydmd}.
+
+\subsection{Test problem for DMD-FPM(n)}
+
+To illustrate the performance of this method, a simple 1-D test problem was designed to resemble a slab BWR core.
```
Comment (RLReed) [inline tex/chapters/chapter6.tex]: Is BWR defined previously?

File: tex/chapters/chapter6.tex [hunk 11]
```diff
@@ -0,0 +1,399 @@
...
+This testing case was initially used to verifying the stability of the Discrete Generalized multigroup method by \citet{rahnema_generalized_2008}.
```
Comment (RLReed) [inline tex/chapters/chapter6.tex]:
```suggestion
This testing case was adapted from previous, transport applications \citep{rahnema_generalized_2008}. 
```

File: tex/chapters/chapter6.tex [hunk 12]
```diff
@@ -0,0 +1,399 @@
...
+        \centering
+        \includestandalone[mode=buildnew, width=0.7\linewidth]{tex/figures/core_materials}
+    \end{minipage}
+    \caption{Configuration for the BWR Test Problem\cite{rahnema_generalized_2008}}
+    \label{fig:BWRconfig}
+\end{figure*}
+
+This core configuration had two unique assemblies.  
+Three fuel types were used, including  4.5 \% enriched $UO_2$ , 2.5\% enriched $UO_2$ , and 4.5 \% enriched $UO_2$ with 5 wt\% $Gd_2O_3$.
+Fuel pins for the second problem were 3.2512 cm thick with 1.1176 cm of moderator on each side.
```
Comment (RLReed) [inline tex/chapters/chapter6.tex]:
```suggestion
Fuel pins for this problem were 3.2512 cm thick with 1.1176 cm of moderator on each side.
```

File: tex/chapters/chapter6.tex [hunk 13]
```diff
@@ -0,0 +1,399 @@
...
+The baseline pincell discretization consisted of 18 mesh cells of fuel enclosed by six mesh cells of moderator;therefore, each pincell will includes 30 mesh points.
```
Comment (RLReed) [inline tex/chapters/chapter6.tex]:
```suggestion
The baseline pincell discretization consisted of 18 mesh cells of fuel enclosed by six mesh cells of moderator;therefore, each pincell contained 30 mesh points.
```

File: tex/chapters/chapter6.tex [hunk 14]
```diff
@@ -0,0 +1,399 @@
...
+Boundary conditions on both sides for this case were vacuum.
+
+Also, the the well-studied 2-D C5G7 benchmark were analyzed \cite{oecd_nuclear_energy_agency_benchmark_2003}, in order to verify the performance of algorithm for multi-dimensional problems.
```
Comment (RLReed) [inline tex/chapters/chapter6.tex]:
```suggestion
The final test problem was the well-studied 2-D C5G7 benchmark \cite{oecd_nuclear_energy_agency_benchmark_2003}, which was used to verify the performance of algorithm for multi-dimensional problems.
```

File: tex/chapters/chapter6.tex [hunk 15]
```diff
@@ -0,0 +1,399 @@
...
+The configure of the benchmark is adapted from the benchmark problem presented in \citet{reed_2015}.
```
Comment (RLReed) [inline tex/chapters/chapter6.tex]:
```suggestion
The configuration of the benchmark was adapted from \citet{reed_2015}.
```

File: tex/chapters/chapter6.tex [hunk 16]
```diff
@@ -0,0 +1,399 @@
...
+The configure of a quarter core contains four fuel pin assemblies and five moderator.assemblies is shown in \FIGURE{fig:C5G7_config}.
```
Comment (RLReed) [inline tex/chapters/chapter6.tex]:
```suggestion
The configuration of a quarter core contains four fuel-pin assemblies and five moderator assemblies as shown in \FIGURE{fig:C5G7_config}.
```

File: tex/chapters/chapter6.tex [hunk 17]
```diff
@@ -0,0 +1,399 @@
...
+Each fuel assembly has 17 $\times$ 17 individual pincells, and the configuration of $UO_2$ assemblyis shown in \FIG{fig:UO2_config} and the configuration of MOX assembly is shown in \FIG{fig:MOX_config}.
```
Comment (RLReed) [inline tex/chapters/chapter6.tex]:
```suggestion
Each fuel assembly used 17 $\times$ 17 individual pincells, and the geometry of a $UO_2$ assembly is shown in \FIG{fig:UO2_config}, while that of a MOX assembly is shown in \FIG{fig:MOX_config}.
```

Comment (RLReed) [reply tex/chapters/chapter6.tex]: Is mixed oxide (MOX) defined previously?

File: tex/chapters/chapter6.tex [hunk 18]
```diff
@@ -0,0 +1,399 @@
...
+Here, each pincell is discretized on a 7$\times$ Cartesian mesh. The demensions are shown in \FIG{fig:MOX_config}.
```
Comment (RLReed) [inline tex/chapters/chapter6.tex]:
```suggestion
Here, each pincell is discretized on a 7$\times$7 Cartesian mesh.
The dimensions are shown in \FIG{fig:MOX_config}.
```

Comment (RLReed) [reply tex/chapters/chapter6.tex]: Are you sure this is the correct label?

File: tex/chapters/chapter6.tex [hunk 19]
```diff
@@ -0,0 +1,399 @@
...
+        (0, 0) rectangle (3,3) node[pos=.5] {MOX};
+        \filldraw[xshift=0 cm, yshift=3 cm, fill=red!30!white, draw=black] 
+        (0, 0) rectangle (3,3) node[pos=.5] {MOX};
+        \draw[xshift=9cm,yshift=4.25cm] node[right] 
+        {\rotatebox{-90}{Vacuum}};
+        \draw[yshift=4.25cm] node[left] {\rotatebox{90}{Reflect}};
+        \draw[xshift=3.25cm,yshift=9.5cm] node[right] {{Vacuum}};
+        \draw[xshift=3.25cm, yshift=-.5cm] node[right] {{Reflect}};
+    \end{tikzpicture}
+    \caption{Configuration for Full-Core.  Each square represents the area of a
```
Comment (RLReed) [inline tex/chapters/chapter6.tex]:
```suggestion
    \caption{Configuration for the C5G7 benchmark.  Each square represents the area of a 
```

File: tex/chapters/chapter6.tex [hunk 20]
```diff
@@ -0,0 +1,399 @@
...
+        \foreach \x in {3*1.26, 13*1.26}
+        \foreach \y in {3*1.26, 13*1.26}
+        \filldraw[xshift=\x cm, yshift=\y cm, fill=blue!30!white, 
+        draw=black] (.63,.63) circle (.5) node[pos=.5] {};
+        \foreach \x in {2*1.26, 5*1.26, 8*1.26, 11*1.26, 14*1.26}
+        \foreach \y in {5*1.26, 8*1.26, 11*1.26}
+        \filldraw[xshift=\x cm, yshift=\y cm, fill=blue!30!white, 
+        draw=black] (.63,.63) circle (.5) node[pos=.5] {};
+    \end{tikzpicture}
+    \caption{Configuration for UO$_2$ fuel bundle.  The green represents a
```
Comment (RLReed) [inline tex/chapters/chapter6.tex]:
```suggestion
    \caption{Configuration for a UO$_2$ fuel bundle.  The green represents a 
```

File: tex/chapters/chapter6.tex [hunk 21]
```diff
@@ -0,0 +1,399 @@
...
+        \foreach \x in {3*1.26, 13*1.26}
+        \foreach \y in {3*1.26, 13*1.26}
+        \filldraw[xshift=\x cm, yshift=\y cm, fill=blue!30!white, 
+        draw=black] (.63,.63) circle (.5) node[pos=.5] {};
+        \foreach \x in {2*1.26, 5*1.26, 8*1.26, 11*1.26, 14*1.26}
+        \foreach \y in {5*1.26, 8*1.26, 11*1.26}
+        \filldraw[xshift=\x cm, yshift=\y cm, fill=blue!30!white, 
+        draw=black] (.63,.63) circle (.5) node[pos=.5] {};
+    \end{tikzpicture}
+    \caption{Configuration for MOX bundle.  The light red represents 4.3\% MOX
```
Comment (RLReed) [inline tex/chapters/chapter6.tex]:
```suggestion
    \caption{Configuration for a MOX bundle.  The light red represents 4.3\% MOX 
```

File: tex/chapters/chapter6.tex [hunk 22]
```diff
@@ -0,0 +1,399 @@
...
+        \draw (1.35,.63) -- (1.55,.63) -- (1.45,.63) -- (1.45,.92) 
+        node[right=0.1cm] {0.54 cm} -- (1.45,1.17) -- (1.35,1.17) -- (1.55, 
+        1.17);
+    \end{tikzpicture}
+    \caption{Configuration for pincell.  The circular fuel element had a 
+             radius of 0.54 cm and was homogenized with cladding for this 
+             model.}
+    \label{fig:pin_cell_config}
+\end{figure*}
+Here, the neutron transport equation for both 1-D and 2-D problems are solved by the discrete ordinates method in a Fortran in-house code unotran, and all DMD calculations were also performed using the PyDMD \cite{pydmd}.
```
Comment (RLReed) [inline tex/chapters/chapter6.tex]:
```suggestion
Here, the neutron transport equation for both 1-D and 2-D problems are solved by the discrete ordinates method, and all DMD calculations were performed using the PyDMD \cite{pydmd}.
```

File: tex/chapters/chapter6.tex [hunk 23]
```diff
@@ -0,0 +1,399 @@
...
+An S4 Gauss-Legendre quadrature was used with the diamond difference approximation.
+The reference eigenvalue and eigenvector were computed using full power iteration (i.e., fully converging the scattering source at each eigen iteration).
+The DMD-FPM($n$) algorithm proposed above was implemented using an in-house code, which solves the neutron transport equation using the discrete ordinates method in Fortran, and all DMD calculations were performed using the Python package {\tt PyDMD} \cite{pydmd}.
```
Comment (RLReed) [inline tex/chapters/chapter6.tex]: Repeated sentence?

File: tex/chapters/chapter6.tex [hunk 24]
```diff
@@ -0,0 +1,399 @@
...
+
+
+\section{Results for DMD-PM(n)}
+As mentioned in the previous chapters, the major cost of the power method is from solving the $\mathbf{Ax =b}$, particularly sweeping over each energy group and spatial mesh points for the discrete ordinates method.
+In this case, the computational time is proportional to the number of iterations, the number of iterations is used as the indicator of computational cost in the following comparisons.
```
Comment (RLReed) [inline tex/chapters/chapter6.tex]:
```suggestion
In this case, the computational time is proportional to the number of iterations.
Thus, the number of iterations is used as the indicator of computational cost in the following comparisons.
```

File: tex/chapters/chapter6.tex (hunk 18 above)
Comment (LeidongX) [reply tex/chapters/chapter6.tex]: Good catch

File: tex/chapters/chapter6.tex (hunk 23 above)
Comment (LeidongX) [reply tex/chapters/chapter6.tex]: fixed at 778f9caf3d5f42249026be08502d4992fb10d4a4

**Pull request merged**